website/docs/d/datasource_cloudfunctions_function.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: "google"
+page_title: "Google: google_cloudfunctions_function"
+sidebar_current: "docs-google-datasource-cloudfunctions-function"
+description: |-
+  Get information about a Google Compute Image.
+---
+
+# google\_cloudfunctions\_function
+
+Get information about a Google Cloud Function.
+
+## Example Usage
+
+```hcl
+data "google_cloudfunctions_function" "my-function" {
+  name = "function"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of a Cloud Function.
+
+- - -
+
+* `project` - (Optional) The project in which the resource belongs. If it
+    is not provided, the provider project is used.
+
+* `region` - (Optional) The region in which the resource belongs. If it
+    is not provided, the provider region is used.
+
+## Attributes Reference
+
+In addition to the arguments listed above, the following computed attributes are
+exported:
+
+* `name` - The name of the Cloud Function.
+* `source_archive_bucket` - The source archive bucket.
```

Comment:
Suggestion: "The GCS bucket containing the zip archive which contains the function"